pkgs/development/python-modules/b2sdk/default.nix

```diff
@@ -81,6 +80,6 @@ buildPythonPackage rec {
     homepage = "https://github.com/Backblaze/b2-sdk-python";
     changelog = "https://github.com/Backblaze/b2-sdk-python/blob/v${version}/CHANGELOG.md";
     license = licenses.mit;
-    maintainers = [ ];
+    maintainers = with maintainers; [ pmw ];
```

Comment:
Can you put this change in a separate commit, please.

Reply:
Done

Reply:
Looks like it was just removed, you can re-add it as another commit "b2sdk: add maintainer pmw".

Reply:
Done